enhancements/cluster-api/installing-openshift-natively-via-cluster-api.md

```diff
@@ -0,0 +1,598 @@
+last-updated: 2023-09-19
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+see-also:
+  - "TBD"
+replaces:
+  - "TBD"
+superseded-by:
+  - "TBD"
+---
+
+# Installing OpenShift natively via Cluster API
+
+## Summary
+
+The following enhancement describes how to install an OpenShift 4 cluster using Cluster API native concepts.
+The enhancement assumes that a Cluster API control plane exists[^1][^2] and that it is running the OpenShift related providers described below.
+
+[^1]: The Cluster API control plane is a set of controllers that manage the lifecycle of Cluster API objects.
+[^2]: A separate enhancement (TBD) will detail how the temporary Cluster API control plane will be provided by the installer for cluster 0 installations.
+
+## Motivation
+
+OpenShift 4 currently relies on terraform, embedded within the installer binary, to provision infrastructure for OpenShift clusters in Installer-Provisioned-Infrastructure clusters.
+This approach has a number of drawbacks that have led to a divergent set of installers, complexity in integrating new providers, and a lack of flexibility in how OpenShift is installed[^3].
+
+It is desirable to have a single installation experience for OpenShift that is consistent across all providers, and can be used not only to bootstrap the first cluster, but also to install additional clusters.
+
+Cluster API is a Kubernetes sub-project that provides a declarative API for managing the lifecycle of Kubernetes clusters and the infrastructure behind them.
+It provides a multi-cluster management solution that will enable OpenShift to unify the single cluster and multi-cluster installation experiences.
+
+[^3]: Leading to various UPI clusters and other installer variations.
+
+### User Stories
+
+As an OpenShift administrator, I want to be able to provision my first OpenShift cluster using the same tooling that I use to provision additional clusters, so that I can have a consistent experience.
+
+As an OpenShift administrator, I want to be able to centrally manage the lifecycle and infrastructure of my OpenShift clusters, so that I do not have to delegate infrastructure management to other teams.
+
+As an OpenShift developer, I want to leverage Cluster API to provision OpenShift cluster infrastructure, so that I can focus on the OpenShift specific components and leverage the community around Cluster API for the generic cluster Infrastructure.
+
+As an OpenShift developer, I want to simplify the maintenance of the installer by removing the embedded terraform and platform specific logic, so that the installer is easier to maintain and extend.
+
+As a Hive developer, I want to delegate as much of the cluster lifecycle to Cluster API as possible, so that Hive can focus on the OpenShift specific features.
+
+### Goals
+
+#### Phase 1
+
+1. Leverage an existing Cluster API control plane to provision infrastructure for OpenShift clusters.
```

Comment:
I think the content here is great, but these strike me more as implementation details than goals. Consider moving this to the proposal section, and replacing with goals, which would be things like "Enable day-2 management of infrastructure" (I'm not sure if that is a valid goal for this phase, but is just an example).